explanation/applications.md

```diff
@@ -9,6 +9,20 @@ To run on the Anbox Cloud platform, applications must fulfil the following requi
 
 If your application fulfils these requirements but you are still having issues running it on Anbox Cloud, file a [bug report](https://bugs.launchpad.net/indore-extern/+filebug).
 
+## Possible application status
```

Comment:
Should it be plural?

Reply:
i.e., 'Possible application statuses'?

Reply:
That is a widely debated topic, some sources say "status", some say "statuses" and some say both are fine. I am trying to find a better source to rely on.